locales/en/data-classes.ftl

```diff
@@ -93,7 +93,7 @@ relationship-statuses = Relationship statuses
 religions = Religions
 reward-program-balances = Reward program balances
 salutations = Salutations
-school-grades-(class-levels) = School grades (class levels)
+school-grades--class-levels- = School grades (class levels)
```

Comment:
This (and L41 above) are what I meant about double hyphens and trailing hyphens.

Also curious what would happen if HIBP adds a new data class. How would we detect that and add it to our l10n chain?